resources/testdata/Cargo.toml

```diff
@@ -47,6 +60,14 @@ locales = [
     # - Often the first non-English locale to receive new data in CLDR
     "fr",
 
+    # Thai:
+    # - Complex word breaking
+    "th",
+
+    # Turkish:
+    # - Interesting case-mappings
```

Comment:
This is putting it nicely. Other ‘I’ words you could have used for "interesting" might be "insidious" or "insufferable". It is also nearly "incomprehensible" on the part of Unicode — mind you the Turkish alphabet and language by themselves make perfect sense, but the "ill-advised" re-use of code points for fundamentally different letters in the encoding "inevitably" leads to a nearly "infinite" string of "inappropriate", "inopportune", and even "illegitimate" usage in the wild. Daha fazla diyebilirdim ama İgilizce'de ı harfı içeren kelime olmadığı için zorlanıyorum.

Reply:
And now you dropped Turkish? To my knowledge none of the languages you went for in the latest change have the same kind of casing issues. Turkish is a good test for handling that issue, which only a handful of languages have.

Reply:
readded